test/defaultHttpClientTests.ts

```diff
@@ -163,7 +163,7 @@ describe("defaultHttpClient", function () {
       download.notified.should.be.true;
     });
 
-    it("for blob or stream bodies", async function () {
+    it.skip("for blob or stream bodies", async function () {
       let payload: HttpRequestBody;
       if (isNode) {
```

Comment:
Did we change the logic to detect isNode? https://github.com/Azure/ms-rest-js/issues/338.
Please change it, since storage will be using electron and the runtime will create issues for them.

Reply:
I'll create another PR for that